src/kili/adapters/kili_api_gateway/asset/formatters.py

```diff
@@ -0,0 +1,24 @@
+"""Formatters for assets retrieved from Kili API."""
+
+import json
+from typing import Dict
+
+from kili.domain.types import ListOrTuple
+
+ASSET_JSON_FIELDS = ("jsonMetadata", "labels.jsonResponse", "latestLabel.jsonResponse")
+
+
+def load_asset_json_fields(asset: Dict, fields: ListOrTuple[str]) -> Dict:
+    """Load json fields of an asset."""
+    if "jsonMetadata" in fields:
+        asset["jsonMetadata"] = json.loads(asset.get("jsonMetadata", "{}"))
+    if "labels.jsonResponse" in fields:
+        asset_labels = asset.get("labels", [])
+        for label in asset_labels:
+            if (label.get("jsonResponse")) is not None:
+                label["jsonResponse"] = json.loads(label["jsonResponse"])
```

Comment:
here, both labels and jsonResponse are mandatory fields, so no need to check
```suggestion
        asset_labels = asset.get("labels", [])
        for label in asset_labels:
            label["jsonResponse"] = json.loads(label["jsonResponse"])
```